packages/procps/escape.c.patch

```diff
@@ -0,0 +1,52 @@
+--- procps-3.3.17.orig/proc/escape.c	2021-02-09 17:11:25.000000000 +0700
++++ procps-3.3.17/proc/escape.c	2021-05-12 00:30:51.000000000 +0700
+@@ -128,17 +128,18 @@
+   int my_bytes = 0;
+ 
+ #if (__GNU_LIBRARY__ >= 6) && (!defined(__UCLIBC__) || defined(__UCLIBC_HAS_WCHAR__))
+-  static int utf_init=0;
++  // This require android 26
++  // static int utf_init=0;
+ 
+-  if(utf_init==0){
+-     /* first call -- check if UTF stuff is usable */
+-     char *enc = nl_langinfo(CODESET);
+-     utf_init = enc && strcasecmp(enc, "UTF-8")==0 ? 1 : -1;
+-  }
+-  if (utf_init==1 && MB_CUR_MAX>1) {
+-     /* UTF8 locales */
+-     return escape_str_utf8(dst, src, bufsize, maxcells);
+-  }
++  // if(utf_init==0){
++  //    /* first call -- check if UTF stuff is usable */
++  //    char *enc = nl_langinfo(CODESET);
```

Comment:
Why you are doing this?

langinfo functionality is provided by ndk patches and libandroid-support.